etna/search/tests/test_views.py

```diff
@@ -223,6 +223,361 @@ def test_accessing_page_with_no_params_performs_empty_search(self):
+    bucket_links_html = (
+        '<ul class="search-buckets__list" data-id="search-buckets-list">'
+    )
+    current_bucket_description_html = (
+        '<p class="search-results__explainer search-results__explainer--bucket">'
+    )
+    search_within_option_html = '<label for="id_filter_keyword" class="search-filters__label--block">Search within results:</label>'
+    sort_order_options_html = '<label for="id_sort_by">Sort by</label>'
+    filter_options_html = '<form method="GET" data-id="filters-form"'
+
+    def patch_api_endpoint(self, url: str, fixture_path: str):
+        full_fixture_path = (
+            settings.BASE_DIR + "/etna/search/tests/fixtures/" + fixture_path
+        )
+        fixture_content = ""
+        with open(full_fixture_path) as f:
+            fixture_content = json_module.loads(f.read())
+        responses.add(responses.GET, url, json=fixture_content, status=200)
+
+    def patch_search_endpoint(self, fixture_path: str):
+        self.patch_api_endpoint("https://kong.test/data/search", fixture_path)
+
+    def assertNoResultsMessagingRendered(self, response):
+        self.assertIn(self.no_results_messaging_html, response)
+
+    def assertNoResultsMessagingNotRendered(self, response):
+        self.assertNotIn(self.no_results_messaging_html, response)
+
+    def assertBucketLinksRendered(self, response):
+        self.assertIn(self.bucket_links_html, response)
+
+    def assertBucketLinksNotRendered(self, response):
+        self.assertNotIn(self.bucket_links_html, response)
+
+    def assertCurrentBucketDescriptionRendered(self, response):
+        self.assertIn(self.current_bucket_description_html, response)
+
+    def assertCurrentBucketDescriptionNotRendered(self, response):
+        self.assertNotIn(self.current_bucket_description_html, response)
+
+    def assertSearchWithinOptionRendered(self, response):
+        self.assertIn(self.search_within_option_html, response)
+
+    def assertSearchWithinOptionNotRendered(self, response):
+        self.assertNotIn(self.search_within_option_html, response)
+
+    def assertSortOrderOptionsRendered(self, response):
+        self.assertIn(self.sort_order_options_html, response)
+
+    def assertSortOrderOptionsNotRendered(self, response):
+        self.assertNotIn(self.sort_order_options_html, response)
+
+    def assertFilterOptionsRendered(self, response):
+        self.assertIn(self.filter_options_html, response)
+
+    def assertFilterOptionsNotRendered(self, response):
+        self.assertNotIn(self.filter_options_html, response)
+
+    def assertResultsRendered(self, response):
+        self.assertIn(self.results_html, response)
+
+    def assertResultsNotRendered(self, response):
+        self.assertNotIn(self.results_html, response)
+
+
+class CatalogueSearchEndToEndTest(EndToEndSearchTestCase):
+    test_url = reverse_lazy("search-catalogue")
+
+    @responses.activate
+    def test_no_matches_for_original_search(self):
+        """
```

Comment:
```suggestion
    def test_no_matches_for_search_query_param_q(self):
        """
```
Not sure about specific terminology, but `_original_search` or `original search` seems weird. 
Passed on similar comment when using {# The original/unfiltered search failed to yield results #} 
Similar occurrences elsewhere `_original_search` or `original search`

Reply:
Adjusted in [1073ff6](https://github.com/nationalarchives/ds-wagtail/pull/729/commits/1073ff6f0f9ea500b39e566b08a924313a88f839)